source/guides/guides/debugging.md

```diff
@@ -65,7 +65,11 @@ it("let's me debug like a fiend", function() {
 })
 ```
 
-Ah, that's better! And functionally equivalent. This is just another example of how Cypress seeks elegance and expressivity for the modern web tester. Fewer keystrokes, more power, don't break standard assumptions, and we all win.
+Ah, that's better! And functionally equivalent. The current subject that was yielded by the {% url `cy.get()` get %} is exposed as `subject` so that you can interact with it in the console.
+
+{% img /img/guides/debugging-subject.png Debugging Subject %}
```

Comment:
I ran the image through `optipng` with the default settings so that it's already optimized for the web.